site/gatsby-site/src/components/submissions/SubmissionEdit.js

```diff
@@ -166,15 +166,17 @@ const SubmissionEdit = ({ id }) => {
                   developers:
                     submission.developers === null
                       ? []
-                      : submission.developers.map((item) => item.name || item),
```

Comment:
The submission had a `harmed_party` with an empty name and was causing the Tags component to crash:

<img width="502" alt="image" src="https://github.com/user-attachments/assets/8e6329f7-842e-489e-83f5-58d0a7dadd54">

I'm not a fan of using entity names instead of ids, but this works for now.